lib/web_ui/flutter_js/src/entrypoint_loader.js

```diff
@@ -151,7 +151,13 @@ export class FlutterEntrypointLoader {
       if (this._ttPolicy != null) {
         jsSupportRuntimeUri = this._ttPolicy.createScriptURL(jsSupportRuntimeUri);
       }  
-      const dartModulePromise = WebAssembly.compileStreaming(fetch(moduleUri));
+
+      const stringBuiltinSupported = _detectWasmStringBuiltin();
```

Comment:
dbc: Does this need to use `this._detectWasmStringBuiltin()`? (see e.g. how `_createScriptTag` is invoked)

Reply:
Good catch, fixed.